gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -1446,6 +1446,21 @@ protected void configureConversionUtilities(Project project, SourceSet sourceSet
         project.getLogger().lifecycle("You can use '-PconvertToPdl.reverse=true|false' to change the direction of conversion.");
       }));
     });
+
```

Comment:
Maybe add a comment here explaining the purpose of this task, similar to the comment corresponding to the above task. Other than that, LGTM